sources/packages/web/src/views/aest/student/StudentApplicationExceptions.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator title="Students" subTitle="Exceptions" />
+    </template>
+
```

Comment:
IMO these blank limes in the middle of the HTML are not bringing better readability hence they can be removed.